### ISSUE TITLE
Crashes w/ OptiFine for Minecraft 1.18

### ISSUE THREAD

**Issue by harvanchik** [Chocohead/OptiFabric#506]
OptiFabric version: 1.12.7
OptiFine version: [HD U H3](https://optifine.net/adloadx?f=OptiFine_1.18_HD_U_H3.jar)
Crash report: https://bytebin.lucko.me/WkB9qUU76i

**Comment by harvanchik** [comment]
Workaround that I found is using the preview version of OptiFine.  The current latest preview version is [HD U H3 pre7](https://optifine.net/adloadx?f=preview_OptiFine_1.18_HD_U_H3_pre7.jar).

**Comment by Slooshie1** [comment]
> Workaround that I found is using the preview version of OptiFine. The current latest preview version is [HD U H3 pre7](https://optifine.net/adloadx?f=preview_OptiFine_1.18_HD_U_H3_pre7.jar).

I just installed the newest OptiFine preview but it still isn't working for me

**Comment by PresidentDoggo** [comment]
> Workaround that I found is using the preview version of OptiFine. The current latest preview version is [HD U H3 pre7](https://optifine.net/adloadx?f=preview_OptiFine_1.18_HD_U_H3_pre7.jar).

Worked for me

**Comment by CaseyB** [comment]
I also got the crash with OptiFine for 1.18 but pre7 works for me. I have OptiFabric 1.12.7

**Comment by ghost** [comment]
Please check other open issues before starting a new one, the top 6 topics below this one or so are all the same issue you just posted, Pre7 is the working version for current Optifabric update, you just have to wait. Spamming the issues section won't help.

**Comment by rdev32** [comment]
then what version of optifabric works with optifine HD U H3 ? 
![image](https://user-images.githubusercontent.com/78289548/145659781-a6abd47e-ab07-4000-8033-7d98f3c1e82e.png)
the website says it works with optifine 1.18 but its working with the pre7 branch
i want to use the stable version of optifine not the pre release

**Comment by harvanchik** [comment]
> then what version of optifabric works with optifine HD U H3 ?

None, as far as I know.

**Comment by Nuanulla** [comment]
Duplicate of #499 and/or #497

Don't use H3 if you want to use OptiFine with other Fabric mods. H3 DOES work with OptiFabric, but not additionally with Fabric API and any mods that require the latter.

**Comment by harvanchik** [comment]
> Duplicate of #499 and/or #497
> 
> Don't use H3 if you want to use OptiFine with other Fabric mods. H3 DOES work with OptiFabric, but not additionally with Fabric API and any mods that require the latter.

I have provided an actual crash report where both of these issues have not.  #499 doesn't even have a description.

**Comment by Nuanulla** [comment]
> > Duplicate of #499 and/or #497
> > Don't use H3 if you want to use OptiFine with other Fabric mods. H3 DOES work with OptiFabric, but not additionally with Fabric API and any mods that require the latter.
> 
> I have provided an actual crash report where both of these issues have not. #499 doesn't even have a description.

And #505 provided an easily-read summary as the OP. Point is, there are now 6 posts regarding the same issue: #497, #499, #502, #505, and now #506. Regardless of what you include, you should've made a reply to #497, and everyone else should've just replied there as well in the first place. New issue posts are meant to signify that it's literally a new issue different than a one posted previously. Aggregate all related info under one common thread.

I don't really want to talk much else about this here.
